### PR TITLE
add pipe operator in 'es' (spanish)

### PR DESCRIPTION
**Pull request by davidvanegas7**

### CODE REVIEW

**Comment by setzer777** [comment]
WHAAT???????? why did 171 files change? O.O

**Comment by setzer777** [comment]
Maybe your encoding was changed.

**Comment by pjhampton** [comment]
Hey @davidvanegas7!

Thanks so much for your contribution! We really appreciate it :smile:. Can you please look into the issue @setzer777 brought up with the file changes.

**Comment by setzer777** [comment]
Of course! I'll do it later on the night c: 

And thanks @davidvanegas7 :D I'll also give a quick review to your translation n.n

**Comment by davidvanegas7** [comment]
Sorry for the 171 files  D:
Can you explain how to add just the files I want?
I want to continue doing the part of the Spanish language

**Comment by pjhampton** [comment]
Please don't worry @davidvanegas7! We really appreciate the contribution :heart:. @setzer777 will help you rectify this.

**Comment by setzer777** [comment]
Yup! I didn't meant to sound mean haha I just got super impressed XD Just let me arrive home and I'll help you out n_n. And yeah! It's cool to contribute this kind of projects, feel free to propose new ideas and add content to the repo :)

**Comment by setzer777** [comment]
@davidvanegas7 join us on slack.

**Comment by davidvanegas7** [comment]
Can you invite me this email (david.vanegas.92@gmail.com)?

**Comment by setzer777** [comment]
https://elixir-slackin.herokuapp.com/
join us!

**Comment by setzer777** [comment]
The channel is elixirschool

**Comment by setzer777** [comment]
@davidvanegas7 regarding our conversation in Slack, could you solve that little problem?

**Comment by setzer777** [comment]
Genial! Quedó resuelto :D

File: es/lessons/basics/pipe-operator.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Para este grupo de ejemplos, usaremos el modulo String de elixir.
+
+- Separar Cadenas (Tokenize String)
+
+```shell
+iex> "Elixir language" |> String.split
+["Elixir", "language"]
+```
+
+- Mayusculas a todos los tokens (Uppercase all the tokens)
```
Comment (setzer777) [inline es/lessons/basics/pipe-operator.md]: Mejor pon `Mayúsculas a todos los caracteres.`

File: es/lessons/basics/pipe-operator.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+Para este grupo de ejemplos, usaremos el modulo String de elixir.
+
+- Separar Cadenas (Tokenize String)
+
+```shell
+iex> "Elixir language" |> String.split
+["Elixir", "language"]
+```
+
+- Mayusculas a todas las palabras (Uppercase all the tokens)
```
Comment (setzer777) [inline es/lessons/basics/pipe-operator.md]: `Mayúsculas a todos los caracteres.`

No son palabras D: son caracteres jeje, y ponle acento a `Mayúsculas`

File: es/lessons/basics/pipe-operator.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+```shell
+iex> "Elixir language" |> String.split |> Enum.map( &String.upcase/1 )
+["ELIXIR", "LANGUAGE"]
+```
+
+- Comparar terminacion de una cadena (Check ending)
```
Comment (setzer777) [inline es/lessons/basics/pipe-operator.md]: Acento a `terminación`

File: es/lessons/basics/pipe-operator.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+```shell
+iex> "elixir" |> String.ends_with?("ixir")
+true
+```
+
+## Buenas Practicas
+
+Si la funcion recibe más de 1 parametro, asegurese de usar parentesis.
```
Comment (setzer777) [inline es/lessons/basics/pipe-operator.md]: Acento a función, asegúrese y paréntesis

File: es/lessons/basics/pipe-operator.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+```shell
+iex> "elixir" |> String.ends_with?("ixir")
+true
+```
+
+## Buenas Practicas
+
+Si la funcion recibe más de 1 parametro, asegurese de usar parentesis. 
+Honestamente no importa mucho en elixir, pero es importante para otros programadores que pueden malinterpretar nuestro codigo. Si en el segundo ejemplo removemos los parentesis de `Enum.map/2`, tendriamos la siguiente advertencia.
```
Comment (setzer777) [inline es/lessons/basics/pipe-operator.md]: Acento a paréntesis y tendríamos

File: es/lessons/basics/pipe-operator.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
+---
+layout: page
+title: Operador Pipe 
+category: basics
+order: 6 
+lang: es
+---
+
+El operador pipe `|>` pasa el resultado de una expresión como el primer parametro de otra expresión.
```
Comment (setzer777) [inline es/lessons/basics/pipe-operator.md]: Acento a parámetro.

Perdón u_u me faltó este D:

**Comment by setzer777** [comment]
@pjhampton I have fully checked the translation, it's ok. I'll be helping @davidvanegas7 to squash his commits and then I think we are ready to go. Thanks a lot David! :D

**Comment by pjhampton** [comment]
Awesome, thanks @setzer777. Also, thanks again @davidvanegas7! What's your twitter handle so @doomspork can give you a shoutout?

Get them squash and we'll get this merged today or tomorrow hopefully.

**Comment by setzer777** [comment]
@pjhampton @doomspork Squash succeeded and the content is verified :+1: :shipit:

**Comment by doomspork** [comment]
:boom: 

Awesome!  Thanks @davidvanegas7 and @setzer777 for all your help

**Comment by setzer777** [comment]
Trabajaba en Intel haha pero no está tan como suena...

**Pull request merged**